src/session.coffee

```diff
@@ -30,6 +30,8 @@ define [
           @_extractAnalyticsSessionOnCreate()
 
         connect: (shop_code)->
+          return console?.warn?('Connect called with invalid shop code') unless shop_code
```

Comment:
should we use `Connect called without a shop code` instead of `Connect called with invalid shop code`?

Reply:
+1 ^